sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/InsertAdaptiveSparkPlan.scala

```diff
@@ -41,9 +41,6 @@ case class InsertAdaptiveSparkPlan(
 
   private val conf = session.sessionState.conf
 
-  // Subquery-reuse is shared across the entire query.
-  private val subqueryCache = new TrieMap[SparkPlan, BaseSubqueryExec]()
```

Comment:
we can ermove the import right?

Reply:
No, unfortunately we can't, `stageCache` still needs it.